File: Javascript/Object Oriented Piracy.js
// https://www.codewars.com/kata/object-oriented-piracy/train/javascript

function Ship(draft,crew) {
    this.draft = draft;
    this.crew = crew;
}
   
Ship.prototype.isWorthIt  = function () {
    return this.draft - this.crew * 1.5 > 20;
}

File: C#/Object Oriented Piracy.cs
// https://www.codewars.com/kata/object-oriented-piracy/train/csharp

public class Ship
{
  public int Draft;
  public int Crew;
  
  public Ship(int draft, int crew)
  {
    Draft = draft;
    Crew = crew;
  }
  
  public bool IsWorthIt() 
  {
    return Draft - (Crew * 1.5) > 20;
  }
}